src/app/calculator/process-cooling/cooling-tower-basin/cooling-tower-basin.component.ts

```diff
@@ -66,11 +70,22 @@ export class CoolingTowerBasinComponent implements OnInit {
     });
     this.hasWeatherBinsDataSub = this.coolingTowerBasinService.hasWeatherBinsData.subscribe(value => {
       this.hasWeatherBinsData = value;
+      if(!this.hasWeatherBinsData){
+        this.isShowingWeatherResults = false;
+      }
     });
+    this.weatherBinSub = this.weatherBinService.inputData.subscribe(value => {
+      let getCaseLength = this.weatherBinService.inputData.getValue().cases.length;
+      if(getCaseLength < 1){
+        this.isShowingWeatherResults = false;
```

Comment:
The form.ts shouldn't still be showing the blue info box when this condition is true. Do '        this.coolingTowerBasinService.isShowingWeatherResults.next(false);' so it publishes/emits to the subscription in form.ts and updates the view